koku/providers/test/aws/test_aws_provider.py

```diff
@@ -255,7 +255,7 @@ def test_check_cost_report_access_fail(self, mock_boto3_client):
     @patch("providers.aws.provider._check_s3_access", return_value=True)
     @patch("providers.aws.provider._check_cost_report_access", return_value=True)
     def test_cost_usage_source_is_reachable(
-        self, mock_get_sts_access, mock_check_s3_access, mock_check_cost_report_access
+        self, mock_check_cost_report_access, mock_check_s3_access, mock_get_sts_access
```

Comment:
These were all passing because we're not using these args in the func. We could replace these with `*args` instead.